shimmingtoolbox/b0map.py

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
This should have a docstring
```suggestion
"""
Compute b0 maps from ...

See: arxiv.org/whatever, plosone.org/whatever, etc
```